src/app/courses/add-courses/courses-add.component.html

```diff
@@ -1,118 +1,132 @@
+    </mat-form-field>
     <planet-form-error-messages [control]="courseForm.controls.description"></planet-form-error-messages>
   </div>
   <div class="row">
     <div class="form-group col-4">
-      <label>Method</label>
-      <input type="text" class="form-control" formControlName="method">
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Method" formControlName="method">
+      </mat-form-field>
     </div>
   </div>
   <div class="row">
     <div class="form-group col-4">
-      <label>Grade Level</label>
-      <select class="form-control" formControlName="gradeLevel">
-        <option *ngFor="let grade of gradeLevels" [value]="grade">{{grade}}</option>
-      </select>
+      <mat-form-field>
+        <mat-select i18n-placeholder placeholder="Grade Level" formControlName="gradeLevel">
+          <mat-option *ngFor="let grade of gradeLevels" [value]="grade"> {{grade}} </mat-option>
+        </mat-select>
+      </mat-form-field>
     </div>
     <div class="form-group col-4">
-      <label>Subject Level</label>
-      <select class="form-control" formControlName="subjectLevel">
-        <option *ngFor="let sub of subjectLevels" [value]="sub">{{sub}}</option>
-      </select>
+      <mat-form-field>
+        <mat-select i18n-placeholder placeholder="Subject Level" formControlName="subjectLevel">
+          <mat-option *ngFor="let sub of subjectLevels" [value]="sub"> {{sub}} </mat-option>
+        </mat-select>
+      </mat-form-field>
     </div>
   </div>
   <div class="row">
     <div class="form-group col-4">
-      <label>Start Date</label>
-      <input type="date" class="form-control" formControlName="startDate">
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Start Date" type="date" formControlName="startDate">
+      </mat-form-field>
       <planet-form-error-messages [control]="courseForm.controls.startDate"></planet-form-error-messages>
     </div>
     <div class="form-group col-4">
-      <label>End Date</label>
-      <input type="date" class="form-control" formControlName="endDate">
+      <mat-form-field>
+        <input matInput type="date" i18n-placeholder placeholder="End Date" formControlName="endDate">
+      </mat-form-field>
       <planet-form-error-messages [control]="courseForm.controls.endDate"></planet-form-error-messages>
     </div>
     <div class="form-group radio col-4">
-      <div>Frequency</div>
+      <div i18n>Frequency</div>
       <!-- radio buttons need same name to allow only one value to be selected! -->
-      <label>
-        <input type="radio" name="daysToggler" (change)="toogleWeekly(true)"> Daily
-      </label>
-      <label>
-        <input type="radio" name="daysToggler" (change)="toogleWeekly(false)"> Weekly
-      </label>
+      <mat-radio-group>
+          <mat-radio-button value = "" name="daysToggler" (change)="toogleWeekly(true)" i18n> Daily </mat-radio-button>
```

Comment:
Here,value is empty. If it is not being use then it should remove otherwise pass the value.